app/scripts/TrackRenderer.jsx

```diff
@@ -1024,14 +1362,17 @@ class TrackRenderer extends React.Component {
     return updated;
   }
 
+  /** @param {string[]} trackIds */
   removeMetaTracks(trackIds) {
     trackIds.forEach((id) => {
       this.metaTracks[id].trackObject.remove();
+      // @ts-expect-error - We are deleting the track object here
       this.metaTracks[id] = undefined;
```

Comment:
I don't this this is strictly necessary if we `delete` the key in the next line, which will also remove references to the object.